### PR TITLE
MacBook Pro 2018 i7 32GB after Apple throttle patch

### PR DESCRIPTION
**Pull request by MaxZheleznyy**
MacBook Pro (15-inch, 2018) with 2.2 GHz Intel Core i7 and 32 GB 2400 MHz DDR4 and Intel UHD Graphics 630 1536 MB
Done everything as mentioned in instructions (5 times fresh build and 5 times incremental builds)

### CODE REVIEW

**Comment by beeradmoore** [comment]
Your commit says  2.9 GHz i7 but your comment here says 2.2GHz i7. Is the commit meant to read 2.2GHz?

**Comment by MaxZheleznyy** [comment]
@beeradmoore  sorry about that, first public commit can't be perfect :) 
So the commit is wrong. My system is 2.2GHz.

**Comment by ashfurrow** [comment]
@MaxZheleznyy no worries at all – would you mind updating the file? No need to squash history or anything fancy. Thanks!

**Comment by MaxZheleznyy** [comment]
@ashfurrow I believe README.md already updated with my data
Or you talking about something else?

File: Readme.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Xcode 9
 
 🖥 | Computer Model | CPU | RAM | Fresh Build Time | Incremental Build Time | Xcode Version | Date & Commit Hash | New Build System? |
 -- | -------------- | --- | --- | ---------------- | ---------------------- | ---- | ---- | ----------------- |
+💻 | MacBook Pro <br />Retina, 15", Mid 2018 | 2.9 GHz i7 | 32 GB | 0:42 | 0:09 |  | 2018-05-19 ([commit](https://github.com/artsy/eidolon/commit/e90a1ffd38d73179a6fe2174587dcd98988efc48)) | :heavy_check_mark:
```
Comment (ashfurrow) [inline Readme.md]: This still lists 2.9 GHz, is that correct?

**Comment by MaxZheleznyy** [comment]
Now I got it :) 
Changed to 2.2

**Review by ashfurrow — APPROVED** [review]

**Pull request merged**